packages/tinacms/src/admin/pages/CollectionCreatePage.tsx

```diff
@@ -240,7 +240,14 @@ const RenderForm = ({ cms, collection, templateName, mutationInfo }) => {
           navigate(`/collections/${collection.name}`)
         } catch (error) {
           console.error(error)
-
+          const defaultErrorText = 'There was a problem saving your document.'
+          if (error.message.includes('already exists')) {
```

Comment:
Down the road might be nice if the backend includes a specific code so that we can make this check a bit more future-proof, but I think this works fine for now.